src/StatusQ/Controls/StatusFloatingButtonsSelector.qml

```diff
@@ -42,6 +42,14 @@ import StatusQ.Popups 0.1
 Row {
     id: floatingButtons
 
+    /*!
+       \qmlproperty repeater
+        This property represents the repeater of selectable items shown to the user.
+        Can be used to assign objectName to the repeater
+        \endqml
+    */
+    property alias repeater: itemSelectionRepeater
```

Comment:
Nitpick: should be `readonly` unless you really want to replace the repeater with your own :)